src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -90,53 +91,59 @@ internal ReflectTypeDescriptionProvider()
         {
         }
 
-        private static Hashtable InitIntrinsicTypeConverters()
-        {
-            // Add the intrinsics
-            return new Hashtable
-            {
-                [typeof(bool)] = typeof(BooleanConverter),
-                [typeof(byte)] = typeof(ByteConverter),
-                [typeof(SByte)] = typeof(SByteConverter),
-                [typeof(char)] = typeof(CharConverter),
-                [typeof(double)] = typeof(DoubleConverter),
-                [typeof(string)] = typeof(StringConverter),
-                [typeof(int)] = typeof(Int32Converter),
-                [typeof(short)] = typeof(Int16Converter),
-                [typeof(long)] = typeof(Int64Converter),
-                [typeof(float)] = typeof(SingleConverter),
-                [typeof(UInt16)] = typeof(UInt16Converter),
-                [typeof(UInt32)] = typeof(UInt32Converter),
-                [typeof(UInt64)] = typeof(UInt64Converter),
-                [typeof(object)] = typeof(TypeConverter),
-                [typeof(void)] = typeof(TypeConverter),
-                [typeof(DateTime)] = typeof(DateTimeConverter),
-                [typeof(DateTimeOffset)] = typeof(DateTimeOffsetConverter),
-                [typeof(Decimal)] = typeof(DecimalConverter),
-                [typeof(TimeSpan)] = typeof(TimeSpanConverter),
-                [typeof(Guid)] = typeof(GuidConverter),
-                [typeof(Uri)] = typeof(UriTypeConverter),
-                [typeof(Color)] = typeof(ColorConverter),
-                [typeof(Point)] = typeof(PointConverter),
-                [typeof(Rectangle)] = typeof(RectangleConverter),
-                [typeof(Size)] = typeof(SizeConverter),
-                [typeof(SizeF)] = typeof(SizeFConverter),
-                // Special cases for things that are not bound to a specific type
-                //
-                [typeof(Array)] = typeof(ArrayConverter),
-                [typeof(ICollection)] = typeof(CollectionConverter),
-                [typeof(Enum)] = typeof(EnumConverter),
-                [s_intrinsicNullableKey] = typeof(NullableConverter),
-            };
-        }
+        private static Hashtable EditorTables => LazyInitializer.EnsureInitialized(ref s_editorTables, () => new Hashtable(4));
```

Comment:
The (new) alternative `Lazy<T>.Value` is [here](https://github.com/manofstick/coreclr/blob/9a8780b414fae62bb831708c62db84effbc405bd/src/mscorlib/src/System/Lazy.cs#L515) where  `_state` is a volatile member.